src/Microsoft.ComponentDetection.Orchestrator/Services/BcdeScanExecutionService.cs

```diff
@@ -39,6 +39,7 @@ public async Task<ScanResult> ExecuteScanAsync(IDetectionArguments detectionArgu
 
         var detectorRestrictions = this.GetDetectorRestrictions(detectionArguments);
         var detectors = this.detectorRestrictionService.ApplyRestrictions(detectorRestrictions, this.detectors).ToImmutableList();
```

Comment:
nit: I know this isn't your variable but if we're going to be using `this.detectors` and `detectors` in the same function we should probably rename it to something like `restrictedDetectors`